test/plugin_functional/plugins/kbn_top_nav/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@lizozom just want to know your opinion. should we provide `tsconfig.json ` and `package.json` for `test/plugin_functional/plugins/**` plugins? 

it looks a little odd to me

Reply:
Well, if things work without them, I guess no :) @alexwizp